[DE] Add `Help_Centre\Account_Restrictions` (#6564)
ppy/osu-wiki: wiki/Help_Centre/Account_Restrictions/de.md

- Chat im Spiel
- Private Nachrichten
- Beiträge in Foren oder Kommentarsektionen
- Beatmaps hochladen oder aktualisieren
- Profildetails aktualisieren
- Einkäufe im Shop

## Mein Account wurde eingeschränkt! Was mache ich?

Wenn dein Account eingeschränkt wurde, verfalle nicht in Panik. Folge bitte diesen Schritten:

- **Warte 24 Stunden.** Einige Einschränkungen sind das Ergebnis automatischer Prozesse. Bitte warte 24 Stunden, da wir täglich jede neue Einschränkung überprüfen - wenn deine Einschränkung ein Irrtum war, wird sie hoffentlich innerhalb dieses Zeitrahmens aufgehoben.
- **Kontaktiere unser [Account-Support-Team](/wiki/People/The_Team/Account_support_team).** Wenn dein Account nach dieser Dauer immer noch eingeschränkt ist, sende bitte eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh) **von der E-Mail-Adresse, die mit deinem Account verknüpft ist,** um nach Details zu deiner Einschränkung zu fragen. Stelle sicher, dass du deinen Benutzernamen sowie jedes Detail, warum du glaubst, dass dein Account eingeschränkt wurde, der E-Mail hinzufügst. Je mehr du uns erzählst, desto leichter ist es für uns, das Problem zu lösen.

> - Wenn du überzeugt bist, dass deine Einschränkung ein Irrtum ist, dann kontaktiere uns bitte sofort, damit wir deinen Fall besprechen können.

Reviewer: ```suggestion
- Wenn du der festen Überzeugung bist, dass deine Einschränkung ein Irrtum ist, dann kontaktiere uns bitte sofort, damit wir deinen Fall besprechen können.
```

Wenn dein Account aufgrund von [Regelbruch](/wiki/Rules) eingeschränkt wurde, wird eine Auszeit von drei Monaten gesetzt, bis du wieder eine Beschwerde einlegen darfst.

Wenn dein originaler Account nicht eingeschränkt ist, dann darfst du dich dort einloggen und normal weiterspielen, aber stelle sicher, dass du ab diesem Zeitpunkt nicht mehr in deine Multi-Accounts einloggst. Machst du es trotzdem, wird das zu härteren Strafen führen.

> Wenn deine Beschwerde zugelassen wurde und dein Account nach dem Spielen erneut eingeschränkt wird, gerate nicht in Panik! Automatisierte Systeme könnten dein Account fälschlicherweise gekennzeichnet, aber dein Account wird innerhalb von 24 Stunden wieder entsperrt, falls das der Fall sein sollte. Sende uns gerne eine E-Mail oder einen Tweet, um unsere Aufmerksamkeit zu gewinnen und schneller entsperrt zu werden!

Reviewer: ```suggestion
Wenn deine Beschwerde zugelassen wurde und dein Account nach dem Spielen erneut eingeschränkt wird, gerate nicht in Panik! Automatisierte Systeme könnten deinen Account fälschlicherweise gekennzeichnet haben, aber dein Account wird innerhalb von 24 Stunden wieder entsperrt, falls das der Fall sein sollte. Sende uns gerne eine E-Mail oder einen Tweet, um unsere Aufmerksamkeit zu gewinnen und schneller entsperrt zu werden!
```

## Warum wurde mein Account eingeschränkt?

Es gibt mehrere Gründe, warum dein Account eingeschränkt werden kann.

> - **Es ist nicht dein erster Account.** Multi-Accounting ist gegen die Regeln. Jeder andere Account neben dem ersten des Spielers wird dauerhaft gesperrt, aber ihr originaler wird in den meisten Fällen unberührt bleiben und der Spieler wird dazu aufgemuntert, auf diesem weiter zu spielen. Wenn das Multi-Accounting häufig auftritt, wird der originale Account als Warnung stumm geschalten und der Hauptaccount wird eingeschränkt, sollte das Verhalten dennoch anhalten. Wenn dein Account eingeschränkt ist und du andere besitzt, bitte greife nicht auf diese zu, sondern nutze ausschließlich den originalen Account. Wenn du Probleme beim Zugriff oder Verifizieren deines Accounts hast, sende uns bitte eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh), damit wir dir helfen können, wieder Zugriff auf deinen Account zu bekommen.

Reviewer: ```suggestion
- **Es ist nicht dein erster Account.** Multi-Accounting ist gegen die Regeln. Jeder andere Account neben dem ersten des Spielers wird dauerhaft gesperrt, aber der originale Account wird in den meisten Fällen unberührt bleiben und der Spieler wird dazu ermutigt, auf diesem weiter zu spielen. Wenn das Multi-Accounting häufig auftritt, wird der originale Account als Warnung stumm geschaltet und der Hauptaccount wird eingeschränkt, sollte das Verhalten dennoch anhalten. Wenn dein Account eingeschränkt ist und du andere besitzt, bitte greife nicht auf diese zu, sondern nutze ausschließlich den originalen Account. Wenn du Probleme beim Zugriff oder Verifizieren deines Accounts hast, sende uns bitte eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh), damit wir dir helfen können, wieder Zugriff auf deinen Account zu bekommen.
```

> - **Du hast jemandem erlaubt, auf deinem Account zu spielen.** Das Teilen von Accounts ist ebenso gegen die Regeln. Wenn du jemandem, Zugriff auf dein Account gibst, dann wird dieser eingeschränkt. Der Account des Spielers, der deinen Account benutzt hat, wird wahrscheinlich stumm geschalten oder gesperrt. Wenn das ausversehen passiert ist, lass uns das bitte sobald wie möglich durch eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh) wissen.

Reviewer: ```suggestion
- **Du hast jemandem erlaubt, auf deinem Account zu spielen.** Das Teilen von Accounts ist ebenso gegen die Regeln. Wenn du jemandem Zugriff auf dein Account gibst, dann wird dieser eingeschränkt. Der Account des Spielers, der deinen Account benutzt hat, wird wahrscheinlich stumm geschaltet oder gesperrt. Wenn das ausversehen passiert ist, lass uns das bitte sobald wie möglich durch eine E-Mail an [accounts@ppy.sh](mailto:accounts@ppy.sh) wissen.
```

- **Du schummelst.** Die Benutzung eines Drittanbieter-Programms, Makros oder jeglicher anderer Form von unfairem Vorteil wird in einer sofortigen Einschränkung enden.
- **Andere Gründe.** Das Account-Team behält sich das Recht vor, Accounts auch aus anderen Gründen einzuschränken.

Wir bitten Spieler von zu Recht eingeschränkten Accounts einige Zeit zu warten, bevor sie ihre Beschwerde einreichen. Das wird als *Auszeit* bezeichnet und die Dauer variiert abhängig von den gemachten Verstößen.

## Was mache ich während der Auszeit vor dem Einreichen einer Beschwerde?

> - **Habe Spaß am Spiel!** Während dein Account eingeschränkt ist und du eine Auszeit hast, kannst du immer noch das Spiel spielen (solange du auf deinem originalen Account spielst!). Die Scores, die du erreichst werden eingereicht und bleiben dir erhalten nachdem die Einschränkung aufgehoben wurde, sofern dein Spielstand nicht zurückgesetzt wird.

Reviewer: ```suggestion
- **Habe Spaß am Spiel!** Während dein Account eingeschränkt ist und du eine Auszeit hast, kannst du immer noch das Spiel spielen (solange du auf deinem originalen Account spielst!). Die Scores, die du erreichst werden eingereicht und bleiben dir erhalten nachdem die Einschränkung aufgehoben wurde, sofern dein Account nicht zurückgesetzt wird.
```

- **Versichere dich, dass du die Regeln respektierst.** Langfristige Einschränkungen sind Bestrafungen für das Brechen der Regeln. Wenn du weitermachst während der Einschränkung, dann wird die Auszeit für das Einreichen einer Beschwerde auf 3 Monate zurückgesetzt. Das beinhaltet beispielsweise weiteres Schummeln, auf den Account anderer Spieler zugreifen oder neue Accounts erstellen, um der Einschränkung zu entgehen.
- **Folge den Anweisungen des Support-Teams.** Die drei monatige Auszeit ist eine Chance, um deine eigenen Aktionen zu reflektieren. Wenn du versucht Beschwerde vor dem Ablaufen der Auszeit einzureichen, werden deine E-Mails ignoriert. Wenn du unsicher bist, wann deine Auszeit ausläuft, und du Beschwerde einlegen darfst oder auf welchem Account du spielen solltest (und die Beschwerde aus einreichen), darfst du uns gerne eine E-Mail senden, um zu fragen.
- **Zögere nicht uns zu kontaktieren, wenn du irgendwelche Fragen hast.** Solange du nicht versuchst, die Strafe zu umgehen, werden wir dir gerne jede Frage über die Einschränkung deines Accounts beantworten!

## Meine Auszeit ist vorbei. Wie lege ich Beschwerde ein?

> Du kannst durch eine E-Mail mit dem Nutzernamen deines Accounts an [accounts@ppy.sh](mailto:accounts@ppy.sh) eine Beschwerde einreichen. Stelle sicher, dass du die E-Mail von der E-Mail-Adresse aus sendest, die mit deinem Account verknüpft ist. Erwarte eine Antwort innerhalb einer Woche nachdem du die Beschwerde eingereicht hast. Wenn du danach noch keine Antwort erhalten hast, dann kontaktiere uns bitte via Twitter oder E-Mail - es gibt die Möglichkeit, dass wir sie nicht erhalten haben.

Reviewer: ```suggestion
Du kannst durch eine E-Mail mit dem Nutzernamen deines Accounts an [accounts@ppy.sh](mailto:accounts@ppy.sh) eine Beschwerde einreichen. Stelle sicher, dass du die E-Mail von der E-Mail-Adresse aus sendest, die mit deinem Account verknüpft ist. Erwarte eine Antwort innerhalb einer Woche nachdem du die Beschwerde eingereicht hast. Wenn du danach noch keine Antwort erhalten hast, dann kontaktiere uns bitte via Twitter oder E-Mail - es besteht die Möglichkeit, dass wir sie nicht erhalten haben.
```

Beim Einreichen einer Beschwerde erwarten wir die folgenden Punkte:

- **Geschichte deines Accounts.** Es muss nicht zu detailiert sein. Das hier sollte alle Verstöße gegen wichtige Vorschriften beinhalten (Multi-Accounting, das Teilen von Accounts, Cheaten oder Stummschaltungen). Du musst nicht an jedes Datum, jede Stummschaltung oder jeden Account, den du erstellt hast, denken, solange du mit deinen Taten ehrlich bist. Dies beinhaltet jede Regel, die du während der Einschränkung gebrochen hast, sofern anwendbar.
- **Warum du die Regeln gebrochen hast.** Eine Erklärung, was dich dazu geführt hat, neue Accounts zu erstellen, deinen Account weiter zu geben, zu schummeln, usw. wird uns helfen (und dir) zu bestimmen, ob die Gefahr besteht, dass du einen weiteren Verstoß begehst. Wenn du während einer Einschränkung Verstöße begehst, dann sind wir ebenso daran interessiert zu wissen, warum.

> - **Wie du die Möglichkeiten erhalten hast.** Das bezieht sich am meisten auf Cheaten - wir wollen wiseen, wie du an die benutzten Tools gelangt bist, wie du darauf gestoßen bist und ob sich die Tools noch auf deinem Computer befinden.

Reviewer: ```suggestion
- **Wie du die Möglichkeiten erhalten hast.** Das bezieht sich am meisten auf Cheaten - wir wollen wissen, wie du an die benutzten Tools gelangt bist, wie du darauf gestoßen bist und ob sich die Tools noch auf deinem Computer befinden.
```

- **Warum wir dich zurücklassen sollten.** Meistens bezieht sich das auf den zweiten Punkt. Wir wollen sicher gehen, dass du die Regeln nicht noch einmal brichst! Bitte verstehe, dass wir mehr in deiner Beschwerde erwarten, wenn es viele Verstöße gab.

## Was, wenn meiner Beschwerde Recht gegeben wird?

> Wenn wir dich zurück lassen, dann musst du eine Einverständnis auf einige Bedingungen geben, bevor die Einschränkung aufgehoben wird.

Reviewer: ```suggestion
Wenn wir uns entscheiden, dich wieder zuzulassen, werden wir verlangen, dass du einigen Bedingungen zustimmst, bevor wir dein Konto wieder freigeben.
```
I think this reads better

- **Du wirst keine Verstöße mehr begehen.** Das beinhaltet Cheaten und das Teilen von Accounts.
- **Du wirst keine weiteren Accounts erstellen, sei es für dich selbst oder jemand anderen.**
- **Du solltest dein Account nicht mit anderen Konten verknüpfen**, einschließlich das Spielen auf gemeinsam genutzten Computern.

> - **Jedes Fehlverhalten wird in einer sofortigen Einschränkung enden**. Das beinhaltet Stummschaltungen.

Reviewer: ```suggestion
- **Jedes Fehlverhalten wird in einer sofortigen Einschränkung enden**. Dies beinhaltet schwerwiegende Stummschaltungen.
```
not just any silence will result in a restriction xd

Zusätzlich erlauben wir keine Änderung der Flagge von deinem Account für ein Jahr.

> In some cases, an appeal returnee may be banned from official and badged tournaments, both as a competitor and member of the tournament's team. Die Dauer der Sperrung ist 1 Jahr pro Verstoß (wenn der Nutzer zum zweiten Mal Beschwerde eingelegt hat, dauert die Sperrung 2 Jahre). Die E-mail zur Entsperrung wird die Dauer der Sperrung darlegen.

Reviewer: ```suggestion
In einigen Fällen kann ein wieder zugelassener Spieler von offiziellen und mit einem Abzeichen versehenen Turnieren ausgeschlossen werden, sowohl als Teilnehmer als auch als Mitglied des Organisationsteams.  Die Dauer der Sperrung ist 1 Jahr pro Verstoß (wenn der Nutzer zum zweiten Mal Beschwerde eingelegt hat, dauert die Sperrung 2 Jahre). Die E-mail zur Entsperrung wird die Dauer der Sperrung darlegen.
```
`wieder zugelassener Spieler` is the best i came up with for `appeal returnee`

Alle E-Mails, die zu der `appeal-review`-Adresse gesendet werden, werden zur Überarbeitung oder weiterer Überprüfung gekennzeichnet und durch eine dritte Partei angeschaut, die nicht in das Tagesgeschäft des Support-Teams involviert ist.

Bitte habe Verständnis, dass das Senden einer E-Mail an diese Adresse nur eine Garantie auf einen zweiten Blick/eine Antwort auf deine Beschwerde gibt. Der Ausgang der Beschwerde kann nicht garantiert werden, sofern du die Regeln innerhalb des Prozesses nicht eingehalten hast.

**Wir werden nur auf Überprüfungen von Beschwerden antworten, die via [appeal-review@ppy.sh](mailto:appeal-review@ppy.sh) gesendet wurden.** Bitte versuche nicht, die Mitglieder des Support-Teams (oder Mitglieder der ehrenamtlichen Teams) via Twitter, Discord, oder jeglicher anderer Plattform zu kontaktieren, um diese Richtlinie zu umgehen - sie können dir nicht helfen.

## Häufige Gründe für Einschränkungen und Auszeiten

| Grund der Einschränkung | Dauer der Auszeit | Hinweise zur Beschwerde |
| :-- | :-- | :-- |
| Beurteilungsfehler | Keine Auszeit | Sofern als irrtümlich erwiesen, hat das keinen Einfluss auf den Account. |

> | Multi-Account | Dauerhaft (Multis) | Hauptaccount wird uneingeschränkt bleiben, möglicherweise stummgeschalten. |

Reviewer: ```suggestion
| Multi-Account | Dauerhaft (Multis) | Hauptaccount wird uneingeschränkt bleiben, möglicherweise stummgeschaltet. |
```

| Exzessives Multi-Accounting | 3 Monate | Beschwerdebedingungen treffen zu, Sperrung von Wettbewerben für ein Jahr |
| Teilen des Accounts | 3 Monate | Beschwerdebedingungen treffen zu, Sperrung von Wettbewerben für ein Jahr, und teilweise Rücksetzung |
| Cheaten | 3 Monate | Beschwerdebedingungen treffen zu, Sperrung von Wettbewerben für ein Jahr, und vollständige Rücksetzung |
| Exzessives Fehlverhalten | Variabel | Beschwerdebedingungen treffen zu. Auszeit hängt von dem früheren Verhalten ab. |

> | Cheaten/Teilen für ein Wettbewerb | 6 Monate | Beschwerdebedingungen treffen zu, entsprechende Rücksetzung, dauerhafte Sperrung von Wettbwerben |

Reviewer: ```suggestion
| Cheaten/Teilen für einen Wettbewerb | 6 Monate | Beschwerdebedingungen treffen zu, entsprechende Rücksetzung, dauerhafte Sperrung von Wettbwerben |
```
